ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-roles_path=../
+roles_path=roles/
```

Comment:
Currently code fits recommended directory structure:
* https://galaxy.ansible.com/intro#share

I think this part should not be changed.